extensions/reactor-client/src/test/java/com/aerospike/client/reactor/ReactorInfoTest.java

```diff
@@ -0,0 +1,25 @@
+package com.aerospike.client.reactor;
+
+import com.aerospike.client.reactor.util.Args;
+import org.junit.Test;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+public class ReactorInfoTest extends ReactorTest{
+
```

Comment:
Any test cases for errors for all newly added operations?

Reply:
added test